gbbs/benchmarks/BatchDynamicConnectivity/SkipList/SkipList.h

```diff
@@ -18,7 +18,10 @@ struct SkipList {
 
         using pointers = std::pair<SkipListElement*, SkipListElement*>;
         using height_array = sequence<pointers>;
-        using values_array = sequence<sequence<sequence<std::pair<uintE, uintE>>>>;
+        using edge_type = std::pair<uintE, uintE>;
```

Comment:
Note that if using suffix `_type` on your types, you may want to us it consistently with your other types, too. (You also want to maintain consistency with the existing style.)